router/terraform/ami.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_ami" "stable_coreos" {
```

Comment:
This is required as our ECS instance hosts run CoreOS - this might be better off in a shared stack.